.github/workflows/pythonpublish.yml

```diff
@@ -20,12 +20,11 @@ jobs:
         python-version: '3.x'
     - name: Install dependencies with pip
       run: |
-        python -m pip install --upgrade pip
-        pip install setuptools wheel twine pep517
```

Comment:
Wheel and setuptools are listed in the pyproject.toml build section and, if we "trust" pep517, we don't need to specify them here. Twine is a different beast b/c it is used to upload to PyPI but is not really a build tool. In theory all we need here is the build tool (+twine).